lasagne/layers/pool.py

```diff
@@ -72,39 +72,45 @@ def pool_output_length(input_length, pool_size, stride,
 
 
 class MaxPool1DLayer(Layer):
-
     """
-    This layer performs max pooling over the final dimension
-    of a 3D tensor.
+    1D max-pooling layer
+
+    Performs 1D max-pooling over the trailing axis of a 3D input tensor.
+
+    Parameters
+    ----------
+    incoming : a :class:`Layer` instance or tuple
+        The layer feeding into this layer, or the expected input shape. The
+        output of this layer should be a 3D tensor.
+
+    pool_size : integer or iterable
+        The length of the pooling region
+
+    stride : integer, iterable or None
+        The stride between sucessive pooling regions.
+        If `None`, stride = pool_size.
```

Comment:
I'm not sure to be honest. I think you're even supposed to use double backticks (that's what you're talking about right?). But I think the rules for the actual type specification may be different. Maybe someone with more numpydoc experience can shed some light on this.